Podfile

```diff
@@ -8,38 +8,14 @@ use_frameworks!
 APP_IOS_DEPLOYMENT_TARGET = Gem::Version.new('13.0')
 
 platform :ios, APP_IOS_DEPLOYMENT_TARGET
+workspace "./WordPressKit.xcworkspace"
```

Comment:
<table data-meta="generated_by_pr-check">
  <tbody>
    <tr>
      <td>:no_entry_sign:</td>
      <td width="100%" data-sticky="false"><span data-href="https://github.com/wordpress-mobile/WordPressKit-iOS/blob/24168ae77b4bb0e20553c5ed86121f9dd76cdf8a/Podfile#L11"></span>Style/StringLiterals: Prefer single-quoted strings when you don't need string interpolation or special symbols.</td>
    </tr>
  </tbody>
</table>